test/fixtures/collection.b/library.b/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "library.b",
+  "version": "1.0.0",
+  "description": "Simple SAPUI5 based library",
```

Comment:
is it SAPUI5 or any UI5 based lib?

Reply:
I wonder if we should use the term "SAPUI5" at all in the open source space. But we have it already in the framework config, so its fine for me.